test/schemas/whototell.schema.test.ts

```diff
@@ -0,0 +1,29 @@
+import { ValidationResult } from '@hapi/joi'
+import { assert } from 'chai'
+
+import { WhototellFormModel } from 'app/models/whototell.model'
+import formSchema from 'app/schemas/whototell.schema'
+
+describe('Who To Tell Schema', () => {
+
+  it('should return no errors when data is valid', () => {
+    const validForm: WhototellFormModel = {
+      confirmation: 'understdood',
+    }
+
+    const errors: ValidationResult = formSchema.validate(validForm)
+
+    assert.isUndefined(errors.error)
+  })
+
+  it('should return errors when data has missing properties', () => {
+    const validForm: WhototellFormModel = {
+    }
+
+    const errors: ValidationResult = formSchema.validate(validForm)
+
+    assert.deepEqual(errors.value, {})
+    assert.equal(errors.error?.details.length, 1)
+    assert.equal(errors.error?.details[0].context?.key, 'confirmation')
```

Comment:
Make sure you assert the error message too. The schema defines the error message, therefore it should be tested.

Also, if you are sure that an optional property will be present in TS, use `!` instead of `?`